flask_wtf/csrf.py

```diff
@@ -110,7 +109,7 @@ def validate_csrf(data, secret_key=None, time_limit=None, token_key='csrf_token'
         digestmod=hashlib.sha1
     ).hexdigest()
 
-    return safe_str_cmp(hmac_compare, hmac_csrf)
+    return hmac.compare_digest(hmac_compare, hmac_csrf)
```

Comment:
could you add a link to the pr https://github.com/pallets/werkzeug/pull/2276/files#diff-97d9d852b7ac5531335c7fdcb2b7e445c9d1d2993d02d56f129202fcdfcafbf3L103-L120

and also you'll want to bump the version https://github.com/benchling/flask-wtf/commit/e2ec35a52a78a8c67940190f50f124ad7e87af4b